tools/ctl/ctl.go

```diff
@@ -502,8 +502,22 @@ var runBenchmark = &cobra.Command{
 			return fmt.Sprint(verdicts[i].Target) > fmt.Sprint(verdicts[j].Target)
 		})
 		log.Printf("Triggering rebuilds on executor version '%s' with ID=%s...\n", executor, run)
-		for _, v := range verdicts {
-			fmt.Printf("%s,%s", fmt.Sprint(v.Target), v.Message)
+		switch *format {
+		// TODO: Maybe add more format options, or include more data in the csv?
+		case "csv":
+			for _, v := range verdicts {
+				cmd.OutOrStdout().Write([]byte(fmt.Sprintf("%s,%s\n", v.Target, v.Message)))
```

Comment:
ugh this actually isn't safe since there are cases where Message has commas. this needs to use encoding/csv :/